### ISSUE TITLE
provide interface to lowlight.registerLanguage

### ISSUE THREAD

**Issue by conorhastings** [conorhastings/react-syntax-highlighter#35]
this might make the light build a bit less clunky feeling, basically thinking it can just be named exprort which would allow doing something like

```js
import SyntaxHighlighter, { registerLanguage } from 'react-syntax-highlighter/dist/light';
```

instead of having to import lowlight directly, what do you think @bmathews

**Comment by bmathews** [comment]
Yeah, I like that. Not a fan of requiring libs I don't have in my package.json.

**Comment by conorhastings** [comment]
published as `2.11.0`
